leftwm/src/bin/leftwm-worker.rs

```diff
@@ -10,7 +10,7 @@ fn main() {
 
     log::debug!("starting leftwm-worker");
 
-    let exit_status = panic::catch_unwind(|| {
+    let completed = panic::catch_unwind(|| {
```

Comment:
I guess this was inadvertedly reverted, right?
```suggestion
    let exit_status = panic::catch_unwind(|| {
```